### ISSUE TITLE
save_image manipulates input tensor inplace

### ISSUE THREAD

**Issue by ptrblck** [pytorch/vision#1337]
Reported in forum: [topic](https://discuss.pytorch.org/t/why-do-i-get-washedout-images-when-displaying-make-grid-output/55972/2) by [Shisho_Sama](https://discuss.pytorch.org/u/Shisho_Sama).

Reason:
`save_image` manipulates the input tensor inplace in [this line of code](https://github.com/pytorch/vision/blob/6de158c473b83cf43344a0651d7c01128c7850e6/torchvision/utils.py#L104).

Code snippet to reproduce:
```python
x = torch.randn(10, 1, 100, 100)
grid = torchvision.utils.make_grid(x)
print(grid.min(), grid.max(), grid.mean())
> tensor(-4.4464) tensor(4.4604) tensor(-0.0003)
torchvision.utils.save_image(grid, 'tmp.png')
print(grid.min(), grid.max(), grid.mean())
> tensor(0.) tensor(255.) tensor(48.1135)
```

Working on a copy would solve this issue.

Let me know, if you would be interested in a quick fix adding a `clone()` to the mentioned line of code or if another approach would be preferable.

**Comment by pmeier** [comment]
This was already fixed in #1244.

**Comment by ptrblck** [comment]
Thanks for the information and sorry for not being able to find the PR :/
